src/pages/home/release-notes/index.md

```diff
@@ -7,6 +7,66 @@ Keywords:
 
 # Release notes
 
+## August 20, 2024
+
+### Android BOM 3.3.0
+
+* This BOM ([Bill of Materials](https://central.sonatype.com/artifact/com.adobe.marketing.mobile/sdk-bom)) release includes changes to the following Android extensions.
+
+<Accordion>
+
+<AccordionItem header='Expand'>
+
+| Extension artifact | BOM (3.2.2) | BOM (3.3.0) |
+|-----|-----|-----|
+| **com.adobe.marketing.mobile:assurance** | **3.0.1** | **3.0.3**|
+| **com.adobe.marketing.mobile:campaign** | **3.0.0** | **3.0.1**|
+| **com.adobe.marketing.mobile:campaignclassic** | **3.1.0** | **3.1.1**|
+| **com.adobe.marketing.mobile:messaging** | **3.1.0** | **3.2.0**|
+| **com.adobe.marketing.mobile:notificationbuilder** | **3.0.0** | **3.0.1**|
+| **com.adobe.marketing.mobile:optimize** | **3.0.1** | **3.0.2**|
+| **com.adobe.marketing.mobile:places** | **3.0.0** | **3.0.1**|
+| com.adobe.marketing.mobile:analytics | 3.0.1 | 3.0.1 |
+| com.adobe.marketing.mobile:audience | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:core | 3.1.1 | 3.1.1 |
+| com.adobe.marketing.mobile:edge | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:edgebridge | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:edgeconsent | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:edgeidentity | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:edgemedia | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:identity | 3.0.1 | 3.0.1 |
+| com.adobe.marketing.mobile:lifecycle | 3.0.1 | 3.0.1 |
+| com.adobe.marketing.mobile:media | 3.1.0 | 3.1.0 |
+| com.adobe.marketing.mobile:signal | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:target | 3.0.0 | 3.0.0 |
+| com.adobe.marketing.mobile:userprofile | 3.0.0 | 3.0.0 |
+
+</AccordionItem>
+
+</Accordion>
+
+### Android Campaign Classic 3.1.1
+
+* Removed unused `app_name` string resource to prevent conflicts with resources defined by the app.
+* Add support for 64-bit broadlog id's.
+* Update notification builder dependency to 3.0.1 to resolve issues with out-of-the-box push templates.
+
+### Android Campaign Standard 3.0.1
+
+* Removed unused `app_name` string resource to prevent conflicts with resources defined by the app.
+* Fixed an ACPCampaign datastore migration issue.

```

Comment:
Is this correct? We only use ACP prefix for iOS.

Reply:
@rymorale could you please help out?

Reply:
you are correct, this should be Android Campaign 2.x